### DISCUSSION TITLE
EP2 stuck in bootloader mode

### DISCUSSION THREAD

**Discussion by TrevorCrouch** [ExpressLRS/ExpressLRS#1416] [Q&A]
Just update all but one of my Ep2's.. typically last one didnt want to play nicely with WIFI, so I built using BF passthrough. Had a 'success'. Unsoldered the boot pins. When it boots it just had a solid green light still. I then resoldered the boot pad, and setup with the ST link. Rebuilt with UART flashing method, once again I got 'success'. Unsoldered the boot pads, reconnected to F/C on uart3. Solid green led still? What AM I missing or doing wrong? Any help would be much appreciated. Thank you

**Comment by schugabe** [comment]
What happens if you just power the rx and disconnect rx/tx from the FC?

**Reply by TrevorCrouch** [reply]
Hi, it's the same unfortunately, just the same solid green led....
